dtfs-central-api/src/repositories/tfm-facilities-repo/tfm-facilities.repo.ts

```diff
@@ -402,4 +403,46 @@ export class TfmFacilitiesRepo {
       throw error;
     }
   }
+
+  /**
+   * Delete a portal amendment for a facility in the database
+   * @param deletePortalFacilityAmendmentParams
+   * @param deletePortalFacilityAmendment.facilityId - The facility id
+   * @param deletePortalFacilityAmendment.amendmentId - The amendment id
+   * @param deletePortalFacilityAmendment.auditDetails - The audit details
```

Comment:
This isn't picking up the comments properly - if you hover over the variable names they should show this description
```suggestion
   * @param deletePortalFacilityAmendmentParams.facilityId - The facility id
   * @param deletePortalFacilityAmendmentParams.amendmentId - The amendment id
   * @param deletePortalFacilityAmendmentParams.auditDetails - The audit details
```